### PR TITLE
matroid automorphisms

### PR DESCRIPTION
**Pull request by antonydellavecchia**
added by request, computes the automorphism group of a mtroid

### CODE REVIEW

File: src/Combinatorics/Matroids/matroids.jl [hunk 1]
```diff
@@ -860,3 +860,32 @@ function affine_geometry(r::Int, q::Int; check::Bool=false)
     PG = projective_geometry(r, q; check=check)
     return restriction(PG, Vector(2:(length(PG.groundset)-q)))
 end
+
+@doc Markdown.doc"""
+    automorphism_group(m::Matroid)
+
+Given a matroid `m` return it's automorphism group as a `PermGroup`
```
Comment (thofma) [inline src/Combinatorics/Matroids/matroids.jl]:
```suggestion
Given a matroid `m` return its automorphism group as a `PermGroup`
```

File: test/Combinatorics/Matroids/Matroids.jl [hunk 2]
```diff
@@ -338,5 +338,10 @@
         @test is_minor(N,NN)
         @test !is_minor(NN,N)
         @test !is_minor(fano_matroid(), uniform_matroid(2,4))
-   end
+    end
+
+    @testset "matroid automorphism" begin
+        M = uniform_matroid(3, 5)
+        @test automorphism_group(M) ==
```
Comment (thofma) [inline test/Combinatorics/Matroids/Matroids.jl]: Something is missing here.

Comment (micjoswig) [reply test/Combinatorics/Matroids/Matroids.jl]: The implementation is not entirely correct (wrong action), and it does not reuse enough code that already exists.  @lkastner is on it.

**Review by micjoswig — APPROVED** [review]
OK.

**Pull request merged**